package.json

```diff
@@ -74,6 +74,7 @@
     "lb-ng-doc": "./node_modules/loopback-sdk-angular-cli/bin/lb-ng-doc"
   },
   "scripts": {
+    "postinstall": "node ./scripts/blip.min.js",
```

Comment:
could this be `sl-blip`, and sl-blip be the bin script of a strong-blip dependency?